### PR TITLE
mh_metric: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/mh_metric.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# mh_metric
+
+> Calculate and enforce code metrics for MATLAB or Octave code.
+> More information: <https://misshit.org>.
+
+- Display code metrics for a MATLAB file:
```
Comment (EmilySeville7cfg) [inline pages/common/mh_metric.md:6]:
```suggestion
- Print the code metrics for the specified files:
```

File: pages/common/mh_metric.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`mh_metric {{path/to/file.m}}`
```
Comment (EmilySeville7cfg) [inline pages/common/mh_metric.md:8]:
```suggestion
`mh_metric {{path/to/input_file1.m path/to/input_file2.m ...}}`
```

File: pages/common/mh_metric.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Display code metrics for an Octave file:
```
Comment (EmilySeville7cfg) [inline pages/common/mh_metric.md:10]:
```suggestion
- Print the code metrics for the specified Octave files:
```

File: pages/common/mh_metric.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`mh_metric --octave {{path/to/file.m}}`
```
Comment (EmilySeville7cfg) [inline pages/common/mh_metric.md:12]:
```suggestion
`mh_metric --octave {{path/to/input_file1.m path/to/input_file2.m ...}}`
```

File: pages/common/mh_metric.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Display code metrics for a specific directory:
```
Comment (EmilySeville7cfg) [inline pages/common/mh_metric.md:14]:
```suggestion
- Print the code metrics for the specified directory:
```

File: pages/common/mh_metric.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`mh_metric {{path/to/directory}}`
+
+- Display code metrics for the current directory:
```
Comment (EmilySeville7cfg) [inline pages/common/mh_metric.md:18]:
```suggestion
- Print the code metrics for the current directory:
```

File: pages/common/mh_metric.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`mh_metric`
+
+- Produce a code metrics report in HTML format:
+
+`mh_metric --html {{path/to/file.html}}`
+
+- Produce a code metrics report in JSON format:
+
+`mh_metric --json {{path/to/file.json}}`
```
Comment (EmilySeville7cfg) [inline pages/common/mh_metric.md:28]:
```suggestion
- Print the code metrics report in HTML/JSON format:

`mh_metric --{{html|json}} {{path/to/output_file.html|path/to/output_file.json}}`

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/mh_metric.md (hunk 7 above)
Comment (navarroaxel) [reply pages/common/mh_metric.md:28]: I think this is more clear
```suggestion
- Print the code metrics report in HTML or JSON format:

`mh_metric --{{html|json}} {{path/to/output_file}}`
```

File: pages/common/mh_metric.md (hunk 4 above)
Comment (navarroaxel) [reply pages/common/mh_metric.md:12]:
```suggestion
`mh_metric --octave {{path/to/file1.m path/to/file2.m ...}}`
```

File: pages/common/mh_metric.md (hunk 2 above)
Comment (navarroaxel) [reply pages/common/mh_metric.md:8]:
```suggestion
`mh_metric {{path/to/file1.m path/to/file2.m ...}}`
```

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/common/mh_metric.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+- Print the code metrics for the specified files:
+
+`mh_metric {{path/to/file1.m path/to/file2.m ...}}`
+
+- Print the code metrics for the specified Octave files:
+
+`mh_metric --octave {{path/to/file1.m path/to/file2.m ...}}`
+
+- Print the code metrics for the specified directory:
```
Comment (mfrw) [inline pages/common/mh_metric.md:14]: Should we add `tree`. It seems from the documentation it does a recursive walk of the specified directory.
```suggestion
- Print the code metrics for the specified directory tree:
```

Comment (navarroaxel) [reply pages/common/mh_metric.md:14]: @mfrw is `recursively` more appropriate?

Comment (mfrw) [reply pages/common/mh_metric.md:14]: Yep I think that sounds much better.

Comment (navarroaxel) [reply pages/common/mh_metric.md:14]:
```suggestion
- Print the code metrics for the specified directory recursively:
```

**Review by blueskyson — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**